app/javascript/packs/complaint-links.js

```diff
@@ -14,22 +19,43 @@ const closeForm = function (containerId = "#tta-activity-form") {
   errorMessage.innerText = ""
 }
 
+// TTA open form
+
 document.querySelector("#js-add-tta-report").addEventListener('click', event => {
   event.preventDefault()
-  openForm()
+  openForm("#tta-activity-form", "#tta-report-display-id")
 })
 
 document.querySelector("#tta-report-icon").addEventListener('click', event => {
-  openForm()
+  openForm("#tta-activity-form", "#tta-report-display-id")
 })
 
+// ITAMS open form
```

Comment:
[question] I decided it was probably more readable just to leave the TTA and monitoring blocks as totally separate event listeners, rather than try to create something like `documentQuerySelectorAll("#tta-report-icon, #monitoring-review-icon")...` type code.  What do you think?

Reply:
Yeah, I think the value of clarity in the `openForm` arguments makes it worthwhile to split up